ast-visual-studio-extension/CxExtension/Toolbar/ProjectsCombobox.cs

```diff
@@ -153,19 +158,30 @@ public void OnChangeProject(object sender, SelectionChangedEventArgs e)
 
             cxToolbar.ResultsTreePanel.ClearAll();
 
-            Project selectedProject = (projectsCombo.SelectedItem as ComboBoxItem).Tag as Project;
 
-            SettingsUtils.StoreToolbarValue(cxToolbar.Package, SettingsUtils.toolbarCollection, SettingsUtils.projectIdProperty, selectedProject.Id);
-            
+            SettingsUtils.StoreToolbarValue(cxToolbar.Package, SettingsUtils.toolbarCollection, SettingsUtils.projectIdProperty, selectedProject);
             if (initialized)
             {
                 SettingsUtils.StoreToolbarValue(cxToolbar.Package, SettingsUtils.toolbarCollection, SettingsUtils.branchProperty, string.Empty);
                 SettingsUtils.StoreToolbarValue(cxToolbar.Package, SettingsUtils.toolbarCollection, SettingsUtils.scanIdProperty, string.Empty);
             }
 
-            _ = branchesCombobox.LoadBranchesAsync(selectedProject.Id);
+            _ = branchesCombobox.LoadBranchesAsync(selectedProject);
 
             cxToolbar.ScanButtonByCombos();
         }
+
+        protected override void ResetOthersComboBoxesAndResults()
+        {
+            cxToolbar.BranchesCombo.IsEnabled = false;
```

Comment:
IsEnabled

Reply:
make sure you are setting it to true

Reply:
In LoadBranchesAsync func:
cxToolbar.EnableCombos(true);